index.html

```diff
@@ -9,19 +9,22 @@
     <body> 
       <main id="container">
         <div id="timer">00:00</div>
-        <img src="img/js-badge.svg" alt="1" id = "img0" class = "card">
-        <img src="img/js-badge.svg" alt="2" id = "img1" class = "card">
-        <img src="img/js-badge.svg" alt="3" id = "img2" class = "card">
-        <img src="img/js-badge.svg" alt="4" id = "img3" class = "card">
-        <img src="img/js-badge.svg" alt="5" id = "img4" class = "card">
-        <img src="img/js-badge.svg" alt="6" id = "img5" class = "card">
-        <img src="img/js-badge.svg" alt="7" id = "img6" class = "card">
-        <img src="img/js-badge.svg" alt="8" id = "img7" class = "card">
-        <img src="img/js-badge.svg" alt="9" id = "img8" class = "card">
-        <img src="img/js-badge.svg" alt="10" id = "img9" class = "card">
-        <img src="img/js-badge.svg" alt="11" id = "img10" class = "card">
-        <img src="img/js-badge.svg" alt="12" id = "img11" class = "card">
       </main>
+      <div id="myModal" class="modal">
+        <div class="modal-content">
+          <p>Please, choose a level:</p>
+          <div class="radiocontainer">
+              <input type="radio" name="difficulty" id="Easy"> <label for="Easy">Easy</label>

```

Comment:
Move `label` to the separate line to improve readability
```html
<input type="radio" name="difficulty" id="Easy">
<label for="Easy">Easy</label>
```
There should be no difference because HTML treats multiple spaces and line breaks as single space ` `.
In any case I suggest you not to rely on ` ` (space) as separation between elements. It may vary from system to system and from browser to browser. It's better to use `margin/text-indent/left/etc`